packages/contracts/scripts/bash/1_StartAnvil.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Starts Anvil with mainnet fork explicitly using default accounts
+anvil -f https://cloudflare-eth.com -m 'test test test test test test test test test test test junk'
```

Comment:
This should use our `runner.ts` I referenced above. You can use `npx tsx` or we can just add `tsx` as a dependency to automatically consume and run TypeScript files.